demisto_sdk/commands/content_graph/parsers/pack.py

```diff
@@ -325,6 +339,17 @@ def parse_ignored_errors(self):
         """Sets the pack's ignored_errors field."""
         self.ignored_errors_dict = dict(get_pack_ignore_content(self.path.name) or {})  # type: ignore
 
+    def parse_pack_readme(self):
+        """Sets the pack's readme field."""
+        path = f"{self.path}/README.md"
+        try:
+            with open(path) as f:
+                self.pack_readme = f.read()
+        except Exception:
```

Comment:
```suggestion
        except FileNotFoundError:
```
If you intend to catch only non-existent README files, this would be the best, otherwise we might catch here unwanted exception for no reason